src/ui/navigation.R

```diff
@@ -25,12 +25,14 @@ enableNavLinks <- function(input, output, spectrumFilesState, modelRunState, sur
     enableTabWhen("Run model", function() { spectrumFilesState$anyDataSets() &&
                                             surveyAndProgramData$anyProgramData() &&
                                             surveyAndProgramData$anySurveyData() &&
-                                            surveyAndProgramData$programDataValid() })
+                                            surveyAndProgramData$programDataValid() &&
+                                            surveyAndProgramData$surveyDataValid()})
     enableTabWhen("Advanced outputs", function() { modelRunState$state == "converged" &&
                                                     spectrumFilesState$anyDataSets() &&
                                                     surveyAndProgramData$anyProgramData() &&
                                                     surveyAndProgramData$programDataValid() &&
-                                                    surveyAndProgramData$anySurveyData() })
+                                                    surveyAndProgramData$anySurveyData() &&
```

Comment:
The requirement that there be some survey points is really a validation rule, so I reckon `anySurveyData` should now be rolled into `surveyDataValid` - no need to have 2 separate functions performing different aspects of validation.